tests/Rector/Contrib/PHPUnit/SpecificMethod/AssertCompareToSpecificMethodRector/Correct/correct.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php declare(strict_types=1);
+
+final class MyTest extends \PHPUnit\Framework\TestCase
+{
+    public function test()
+    {
+        $this->assertCount(5, $something);
+        $this->assertNotCount(5, $something, 'third argument');
+        $this->assertInternalType('string', $something);
+    }
+}
```

Comment:
**I love this test case very much** - short, clear and covers all possible combinations!

It makes my :eyes: :tada: